site/pages/experimental-fr.hbs

```diff
@@ -242,3 +248,17 @@ if ( s.getTime() &lt; msT && msT &lt; e.getTime() ) {
 		</div>
 	</div>
 </div>
+
+
+<h3 id="well-header-rwd-bgimg">Icon warning color</h3>
+<p>Adding <code>experimental icon-warning-light</code> in the class will change the color</p>
+<div class="bg-warning">
+	<h1>
+		<span class="glyphicon glyphicon-warning-sign mrgn-rght-md"></span> Without "experimental icon-warning-light" class
+	</h1>
+</div>
+<div class="bg-warning">
+	<h1 class="experimental icon-warning-light">
+		<span class="glyphicon glyphicon-warning-sign mrgn-rght-md"></span> Without "experimental icon-warning-light" class
+	</h1>
```

Comment:
Can you put the class on the icon only and make the h1 a paragraph with class h1 instead? Like so:
```suggestion
	<p class="h1">
		<span class="glyphicon glyphicon-warning-sign experimental icon-warning-light mrgn-rght-md"></span> Without "experimental icon-warning-light" class
	</p>
```
After this, it is good to go!

Reply:
and same for the h1 in the example above without warning color.